sample_factory_examples/dmlab/dmlab_env.py

```diff
@@ -147,11 +150,11 @@ def make_dmlab_env_impl(spec, cfg, env_config, **kwargs):
     return env
 
 
-def make_dmlab_env(env_name, cfg=None, **kwargs):
+def make_dmlab_env(env_name, cfg, env_config, **kwargs):  # TODO: env_config required here for DMlab?
```

Comment:
make_env_func should take 3 arguments.
See the new function `register_env()` and the docstring comments. Also new type CreateEnvFunc used for typehints